### ISSUE TITLE
Feature Request: Leaderboard page showing Users who rated the most movies

### ISSUE THREAD

**Issue by InputEvelution** [TASVideos/tasvideos#864]

**Comment by Masterjun3** [comment]
What does this issue ticket request exactly?
We have a list of movies sorted by ratings here: https://tasvideos.org/Movies-standard-alternative-SortY

**Comment by Noxxaa** [comment]
It also already exists in list form as part of MovieStatistics: https://tasvideos.org/MovieStatistics/HighestAverageRating

I could see merit in something that actually "ranks" the top 100 or so but it would only be marginally different

**Comment by InputEvelution** [comment]
No, you misunderstand, this isn't a page that ranks movies by their highest average rating. I'm referring to the page that existed on the old site which ranked users by how many movies they had rated (showing the top 50 iirc).

**Comment by Masterjun3** [comment]
Ah, got it, that makes a lot more sense as a feature request.
